packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -279,5 +323,112 @@ describe.skip('plugin-calendar', () => {
           });
       });
     });
+    describe('#meetingNotes', () => {
+      let creator, mccoy, webex, spock;
+
+      before('create test users', () => testUsers.create({
+        count: 3,
+        config: {
+          entitlements: [
+            'webex',
+            'squaredCallInitiation',
+            'squaredInviter',
+            'squaredRoomModeration',
+            'webExSquared',
+            'squaredFusionCal'
+          ]
+        }
+      })
+        .then((users) => {
+          console.log('Enter then');
```

Comment:
These logs are helpful during development, but please remove them here.

Reply:
Will do, sorry about that